src/transformers/integrations/peft.py

```diff
@@ -144,11 +147,20 @@ def load_adapter(
                 "You should either pass a `peft_model_id` or a `peft_config` and `adapter_state_dict` to load an adapter."
             )
 
+        # We keep `revision` in the signature for backward compatibility
+        if revision is not None and "revision" not in adapter_hub_kwargs:
+            adapter_hub_kwargs["revision"] = revision
+        elif revision is not None and "revision" in adapter_hub_kwargs:
+            logger.warning(
```

Comment:
Maybe `error` level as this seems like a potential source of hidden error

Reply:
Makes sense, changed it in d3c1cadb2951004911c252f4eb2c19d301140349